grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -81,10 +81,15 @@ class InvoiceItem implements Serializable {
         glAccount(nullable: true)
         budgetCode(nullable: true)
         quantity(nullable: false, min: 0, validator: { Integer quantity, InvoiceItem obj ->
+            // Order adjustments are not quantity-based, they should always have quantity = 1
+            if(obj.orderAdjustment && quantity > 1) {
```

Comment:
Should it be `quantity != 1`?

Also, I feel like a more proper fix for this would be to grey out the quantity field for adjustments on the frontend and not let you edit it at all but this is fine for now.